cmd/clusterctl/client/client.go

```diff
@@ -65,6 +65,15 @@ type Client interface {
 	// ProcessYAML provides a direct way to process a yaml and inspect its
 	// variables.
 	ProcessYAML(options ProcessYAMLOptions) (YamlPrinter, error)
+
+	// Interface for alpha features in clusterctl
+	AlphaClient
+}
+
+// AlphaClient exposes the alpha features in clusterctl high-level client library.
+type AlphaClient interface {
+	// RolloutRestart provides rollout restart of cluster-api resources
+	RolloutRestart(options RolloutRestartOptions) error
```

Comment:
I can see that as other "alpha" commands get added in, this interface could grow large. Since it seems that there are many rollout related functions, we can perhaps embed or add a `Rollout rollout.Client` interface here.

Something similar to `cmd/clusterctl/client/repository/client.go` and `cmd/clusterctl/client/cluster/client.go`.

This way all the rollout related behavior can be kept in that interface and that can be just referenced here.

Reply:
I refactored the code. PTAL

Reply:
I'm not sure how that would work? Rollout looks like this:
```
type Rollout interface {
	ObjectRestarter(cluster.Proxy, util.ResourceTuple, string) error
}
```
Where would the `RolloutRestart(options RolloutRestartOptions) error` reside? 

I believe I had addressed @wfernandes comments with a refactor earlier.